src/Utilities/FlowAnalysis/FlowAnalysis/Framework/DataFlow/DataFlowOperationVisitor.cs

```diff
@@ -4020,11 +4018,6 @@ public sealed override TAbstractAnalysisValue VisitLocalFunction(ILocalFunctionO
         /// </summary>
         protected INamedTypeSymbol? TaskNamedType { get; }
 
-        /// <summary>
-        /// <see cref="INamedTypeSymbol"/> for <see cref="System.IO.MemoryStream"/>
-        /// </summary>
-        protected INamedTypeSymbol? MemoryStreamNamedType { get; }
```

Comment:
Technically these are breaking changes for any third-party sub-types of DataflowOperationVisitor, as can be seen from the build errors. However, we have never guaranteed back compat for this package, so I'll be fine with us deleting the corresponding entries in the public API file OR you can revert the changes to this source file. Either way should be fine.

Reply:
Let me revert, that's fine, it will just not be used but the impact should be minimal.